include/deal.II/distributed/tria.h

```diff
@@ -555,6 +555,11 @@ namespace parallel
       communicate_locally_moved_vertices(
         const std::vector<bool> &vertex_locally_moved);
 
+      /**
+       * @copydoc dealii::Triangulation::subdomain_id()
+       */
+      virtual types::subdomain_id
+      subdomain_id(const CellId &cell_id) const override;
```

Comment:
Maybe call the function `get_owner_of_cell()` to clarify its purpose? Or `get_owning_subdomain_for_cell()`?